dif-data-classes/build.gradle.kts

```diff
@@ -26,6 +32,7 @@ kotlin {
             dependencies {
                 implementation(project.napier())
                 implementation(project.ktor("http")) {
+                    // TODO is this still needed? and why only in this project?
```

Comment:
👀 @JesusMcCloud

Reply:
removed it and successfully exported locally